src/api/status/public/index.html

```diff
@@ -355,66 +303,17 @@ <h4 class="mb-0" id="total-contributors-satellite">&nbsp;</h4>
                     /></a>
                   </div>
                   <div class="d-flex flex-column justify-content-center">
-                    <p class="mb-0">
-                      <a
-                        href=""
-                        title="User"
-                        target="_blank"
-                        id="latest-author-satellite"
-                        class="text-capitalize"
+                    <p class="mb-0 text-capitalize">
+                      <a href="" title="User" target="_blank" id="latest-author-satellite"
                         >&nbsp;</a
                       >
-                      <a
-                        href=""
-                        title="Commit"
-                        target="_blank"
-                        id="latest-author-commit-satellite"
-                        class="font-monospace btn-link text-xs"
-                      ></a>
                     </p>
                   </div>
                 </div>
               </div>
             </div>
           </div>
-          <div class="col-xl-3 col-sm-6 mb-xl-0 mb-4 mt-4">
-            <div class="card card--telescope">
-              <div class="card-header card-header--telescope p-3 pt-2">
-                <div
-                  class="icon icon-lg icon-shape bg-gradient-info shadow-info text-center border-radius-xl mt-n4 position-absolute"
-                >
-                  <i class="material-icons opacity-10">person</i>
-                </div>
-                <div class="text-end pt-1">
-                  <h6 class="mb-0 text-capitalize">Telescope</h6>
-                  <p class="text-sm mb-0 text-capitalize">Total posts</p>
-                  <h4 class="mb-0" id="totalPosts">&nbsp;</h4>
-                </div>
-              </div>
-              <hr class="dark horizontal my-0" />
-              <div class="card-footer p-3"></div>
-            </div>
-          </div>
-          <div class="col-xl-3 col-sm-6 mb-xl-0 mb-4 mt-4">
-            <div class="card card--telescope">
-              <div class="card-header card-header--telescope p-3 pt-2">
-                <div
-                  class="icon icon-lg icon-shape bg-gradient-info shadow-info text-center border-radius-xl mt-n4 position-absolute"
-                >
-                  <i class="material-icons opacity-10">person</i>
-                </div>
-                <div class="text-end pt-1">
-                  <h6 class="mb-0 text-capitalize">Telescope</h6>
-                  <p class="text-sm mb-0 text-capitalize">Total feeds</p>
-                  <h4 class="mb-0" id="feeds-count">&nbsp;</h4>
-                </div>
-              </div>
-              <hr class="dark horizontal my-0" />
-              <div class="card-footer p-3"></div>
-            </div>
-          </div>
```

Comment:
Why are these removed? They are cards for total posts and feeds.

Reply:
Oh, I only removed the li tags in the sidebar. Maybe because my version was not the latest one then...I will add it